src/layers/core/path-layer/path-layer-vertex-64.glsl.js

```diff
@@ -63,20 +63,22 @@ vec3 lineJoin(vec2 prevPoint64[2], vec2 currPoint64[2], vec2 nextPoint64[2]) {
   vec2_sub_fp64(currPoint64, prevPoint64, deltaA64);
   vec2_sub_fp64(nextPoint64, currPoint64, deltaB64);
 
+  vec2 lengthA64 = vec2_length_fp64(deltaA64);
+  vec2 lengthB64 = vec2_length_fp64(deltaB64);
```

Comment:
Decide to calculate the length of the segments around a joint in 64-bit because they are used to decide if any joint is start/end or not